components/arm/trossen/trossen.go

```diff
@@ -111,9 +128,6 @@ func (config *AttrConfig) Validate(path string) error {
 	if config.BaudRate == 0 {
 		return errors.New("expected nonempty serial_baud_rate")
```

Comment:
Is it? trossen gives a range of baudrates for this model. Where did you get that being the only number from?
https://www.trossenrobotics.com/widowx-250-robot-arm.aspx#hardware

Reply:
The servos support all those rates, but to actually _change_ the baudrate, you have to manually go in and modify the settings on each servo yourself, which I wouldn't expect people to do (since 1mbps is plenty fast for everything an arm like this is doing.) So it's effectively a fixed baud rate that Trossen sets in their factory.

https://docs.trossenrobotics.com/interbotix_xsarms_docs/specifications/vx300.html#default-servo-configurations

Reply:
There is a future I can imagine where we can reconfigure things through our driver, but for this PR, I'd be very comfortable omitting all baudrate config fields, and setting the 1mbps default in the trossen arm creation.